modules/sfm/CMakeLists.txt

```diff
@@ -0,0 +1,83 @@
+set(the_description "SFM algorithms")
+
+ocv_warnings_disable(CMAKE_CXX_FLAGS -Wundef -Winvalid-pch)
+
+# needed since we are linking static libs
+set(CMAKE_CXX_FLAGS ${CMAKE_CSS_FLAGS} "-fPIC")
+
+
+### LIBMV LIGHT DEFINITIONS ###
+
+set(LIBMV_LIGHT_INCLUDES
+  src/libmv_light
+  src/libmv_light/third_party/ssba
+)
+
+set(LIBMV_LIGHT_LIBS
+  reconstruction
+  correspondence
+  camera
+  multiview
+  multiview_test_data
+  numeric
+)
+
+
+### LIBMV LIGHT EXTERNAL DEPENDENCIES ###
+
+find_package( Ceres )
```

Comment:
Please use QUIET here (to eliminate CMake warning) and special message below. Something like this:
https://github.com/Itseez/opencv/blob/09b9b0fb9e9c9dd8c9e0d65705f8f19aa4c27f8a/cmake/OpenCVDetectVTK.cmake#L10-L17